policy/aggregation_type.go

```diff
@@ -463,8 +467,8 @@ func (id AggregationID) Contains(aggType AggregationType) bool {
 	if !aggType.IsValid() {
 		return false
 	}
-	idx := int(aggType) >> 6
-	offset := uint(aggType) & 0x3F
+	idx := int(aggType) >> aggIDBitShift   // aggType / 64
```

Comment:
nit: probably don't need extra comment here in case the value of `aggIDBitShift` and `aggIDBitMask` change in the future causing the comment to be outdated.

Reply:
Feels a lot more readable with the comments, I'd prefer to leave it unless you feel strongly